Dockerfiles/midstream/README.md

```diff
@@ -1,6 +1,12 @@
 # operator-bundle-validate-container
 CVP midstream operator bundle validate container
 
+## Prerequisites
+
+In order to run unit_tests.py on your local without any issues, you need to make sure your SELinux is set to `Permissive`.
+To check your current status of SELinux just run command `# getenforce`.
+To achieve Permissive mode you need to run following command `# setenforce 0`
```

Comment:
Please remove such wrong recommendation. There are more and less dangerous ways how to treat SELinux violation.
* Fix SELinuxl labeling on host
* disable SELinux enforcement just for the container

Reply:
Changed, specified command to run on the local to successfully run the unit_tests.py